grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -581,7 +581,11 @@ private void handleResponse(LoadBalanceResponse response) {
         return;
       }
 
-      if (typeCase != LoadBalanceResponseTypeCase.SERVER_LIST) {
+      if (typeCase == LoadBalanceResponseTypeCase.FALLBACK_RESPONSE) {
+        useFallbackBackends();
+        maybeUpdatePicker();
```

Comment:
Do you want to call `cancelFallbackTimer()`?

Reply:
done.